types/spotify-web-playback-sdk/index.d.ts

```diff
@@ -64,23 +79,29 @@ declare namespace Spotify {
     }
 
     interface PlaybackDisallows {
-        pausing: boolean;
-        peeking_next: boolean;
-        peeking_prev: boolean;
-        resuming: boolean;
-        seeking: boolean;
-        skipping_next: boolean;
-        skipping_prev: boolean;
+        pausing?: boolean;
+        peeking_next?: boolean;
+        peeking_prev?: boolean;
+        resuming?: boolean;
+        seeking?: boolean;
+        skipping_next?: boolean;
+        skipping_prev?: boolean;
+        toggling_repeat_context?: boolean;
+        toggling_repeat_track?: boolean;
+        toggling_shuffle?: boolean;
     }
 
     interface PlaybackRestrictions {
-        disallow_pausing_reasons: string[];
-        disallow_peeking_next_reasons: string[];
-        disallow_peeking_prev_reasons: string[];
-        disallow_resuming_reasons: string[];
-        disallow_seeking_reasons: string[];
-        disallow_skipping_next_reasons: string[];
-        disallow_skipping_prev_reasons: string[];
+        disallow_pausing_reasons?: string[];
```

Comment:
could we use an enum here instead of a string? is the list of possible `reasons` known?